mimic/model/maas_objects.py

```diff
@@ -130,6 +130,7 @@ def to_json(self):
                 'label': self.label,
                 'criteria': self.criteria,
                 'check_id': self.check_id,
+                'entity_id': self.entity_id,
```

Comment:
This is a bit of a rider, because it turns out Intelligence tests are still broken without it.